app/controllers/api/did_not_votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class API::DidNotVotesController < API::RestfulController
+
+  def create
+    raise NotImplementedError.new
```

Comment:
I'd be happy to not have these NotImplementedError stubs; so long as we don't have a route in place for create/update/destroy, we won't reach them.